src/parser/warrior/fury/modules/talents/FrothingBerserker.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import Analyzer from 'parser/core/Analyzer';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+import Events from 'parser/core/Events';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+import SUGGESTION_IMPORTANCE from 'parser/core/ISSUE_IMPORTANCE';
+
+/*  Example log:
+ *  https://www.warcraftlogs.com/reports/KhynM7v96cZkTBdg#fight=6&type=damage-done&source=78
+ */
+
+
+const DAMAGE_BONUS = 0.1;
+
+class FrothingBerserker extends Analyzer {
+    static dependencies = {
+        abilityTracker: AbilityTracker,
+    };
+
+    damage = 0;
+
+    constructor(...args) {
+        super(...args);
+
+        this.active = false;
+
+        if (this.selectedCombatant.hasTalent(SPELLS.FROTHING_BERSERKER_TALENT.id)) {
+            this.active = true;
+            this.addEventListener(Events.damage.by(SELECTED_PLAYER), this.onPlayerDamage);
+        }
+    }
+
+    onPlayerDamage(event) {
+        if (this.selectedCombatant.hasBuff(SPELLS.FROTHING_BERSERKER.id)) {
+            this.damage += calculateEffectiveDamage(event, DAMAGE_BONUS);
+        }
+    }
+
+    get damagePercent() {
+        return this.owner.getPercentageOfTotalDamageDone(this.damage);
+    }
+
+    get damageIncreasePercent() {
+        return this.damagePercent / (1 - this.damagePercent);
+    }
+
+    get uptimePercent() {
+        const uptime = this.selectedCombatant.getBuffUptime(SPELLS.FROTHING_BERSERKER.id) / 1000;
+        const fightLengthSec = this.owner.fightDuration / 1000;
+
+        return uptime / fightLengthSec;
+    }
+
+    get suggestionThresholds() {
+        return {
+          isLessThan: {
+            minor: 0.65,
+            average: 0.6,
+            major: 0.55,
+          },
+          style: 'percentage',
+        };
+      }
+
+    suggestions(when) {
+        const {
+            isLessThan: {
+              minor,
+              average,
+              major,
+            },
+          } = this.suggestionThresholds;
+
+        when(true).isTrue().addSuggestion((suggest) => {
```

Comment:
Not sure about other people, but personally im not a huge fan of telling the player its a mistake to use the talent they picked unless one talent is vastly greater than the other. Im not going to tell you to change it cause i have no idea how close these talents actually are .... and its just my personal opinion as someone who believes players should play the way they want to play (if the talents are close). Feel free to leave it in though, I know something similar was added for Rune of Power for mages so its fine to have it. I would say just make sure you are only doing this when it is important and not for every talent that isnt the "Cookie cutter best build"

Reply:
There is absolutely unequivocally no situation in which this talent is an increase in DPS, ever. It's a non-choice. It's not so much cookie cutter as it has "gimping yourself for less DPS and clunkier gameplay". I'd love to see frothing actually be good, but it really isn't. :/

Reply:
Thats fine. Then absolutely leave it in.